src/hooks/useFarmStats.ts

```diff
@@ -15,10 +15,9 @@ const ZERO = 0n;
 
 const graphqlClient = new Client({ url: WPOKT_SUBGRAPH_URL ?? '' });
 
-function buildFarmStatsQuery(farmAddress: string) {
-  return gql`
-  query {
-    tokenGeysers(id: "${farmAddress}") {
+const FARM_STATS_QUERY: any = gql`
```

Comment:
Doesn't this have a type available somewhere? pretty sure `graphql-request`'s written in TS

Reply:
> Doesn't this have a type available somewhere? pretty sure `graphql-request`'s written in TS

afaik `graphql-tag` returns any, type safety was reverted in this PR https://github.com/apollographql/graphql-tag/pull/141.

Reply:
Yea, it returns a `DocumentNode` type.